packages/react/README.md

```diff
@@ -403,8 +403,8 @@ function NotificationsList() {
 
   return (
     <ul>
-      {store.notifications.map((notification) => (
-        <li>{notification.title}</li>
+      {store?.notifications.map((notification) => (
+        <li key={notification.id}>{notification.title}</li>
```

Comment:
These are the only changes I intended! Think it was husky that did the formatting above. Looking into it...

Reply:
It's fine. Someone might have committed once without the pre-commit hooks, and you actually fixing it right now. No worries.